core/lib/spree/app_configuration.rb

```diff
@@ -268,6 +269,16 @@ def available_currencies
       @available_currencies ||= ::Money::Currency.all
     end
 
+    # Allows implementing custom shipment building criterion based on an order
+    # instance. Warning: change this setting is not recommended.
+    # @!attribute [rw] build_shipment_predicate_class
+    #   @note Setting this to false is not recommended. This setting is provided
```

Comment:
Maybe `Setting this to false` is misleading? 
Is this statement referring to the result of the `call` method on the class specified in this preference? If yes maybe this is not the right place where to put this information or it needs to be adjusted?